_posts/2021-03-01-changelog-142.markdown

```diff
@@ -0,0 +1,93 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #142"
+date: 2021-03-01 17:00:00 +0300
+---
+
+
+## New Features
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6877 -->
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6835 -->
+* Debugging is now available in [PyCharm Professional](https://www.jetbrains.com/pycharm/) and
+  [GoLand](https://www.jetbrains.com/go/) using
+  [Native Debugging Support](https://plugins.jetbrains.com/plugin/12775-native-debugging-support) plugin
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6807 -->
+* `Change signature` (`Refactor | Change Signature` or <kbd>Ctrl+F6</kbd>) refactoring now allows you to choose a
+  default value for new parameters (by [@Kobzol])
+
+## Fixes
+
+* Improvements in [new name resolution](https://github.com/intellij-rust/intellij-rust/issues/6217) engine
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/6734 -->
+    * Fix resolve macro calls inside file included by `include!` macro
+
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/6730 -->
+    * Support file level `cfg` attributes
+
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/6721 -->
+    * Support basic multi-resolve of inline modules
+
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/6720 -->
+    * Fix resolve inside cfg-disabled inline `mod`
+
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/6773 -->
+    * Use old resolve in test/bench non-workspace crates
+
+  Note, you can enable the new engine via `Use experimental name resolution engine` option in
+  `Preferences | Languages & Frameworks | Rust settings`
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6874 -->
+* Properly handle default type arguments in `Extract function` refactoring
```

Comment:
Let's add path where this refactoring can be found and its shortcut in the same way as we did with "Change signature"